crates/test-programs/Cargo.toml

```diff
@@ -10,20 +10,24 @@ license = "Apache-2.0 WITH LLVM-exception"
 [build-dependencies]
 cfg-if = "1.0"
 cargo_metadata = "0.15.3"
+wit-component = "0.7.4"
```

Comment:
Could this move to a `{ workspace = true }` dep in the top-level `Cargo.toml` since it's probably gonna show up in a few places? (also good to have in one place to update when all of `wasm-tools` is bumped)